### DISCUSSION TITLE
Custom DisplayVersion of a bundle in ARP

### DISCUSSION THREAD

**Discussion by maciejtoporowicz** [wixtoolset/issues#7206] [Questions] [answered]
Hello, 

In my project I'd like to use a custom `DisplayVersion` of a bundle in ARP. Instead of having a version like `1.2.3.4` I'd like to see some text there like `Version XYZ`.

I can see it's doable if I manually change the `DisplayVersion` registry key `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\{some-guid}`, but I can't find a way to make it happen automatically via a `.wxs` file.

Not sure if it makes sense, but I was expecting to be able to do something like this:
```
<Bundle 
    Name="MyBundleName"
    Version="1.2.3.4"
    DisplayVersion="Version XYZ"
    ....
>
```
However I can't find anything like this in the docs.

I saw https://github.com/wixtoolset/issues/issues/5496 but I can't find the answer to my question there.

I assume a workaround for this is to run a custom script after the bundle is installed which manually changes the registry key, but it feels like an unnecessary hack. Is there another way to do it?

**Comment by barnson** [comment, marked as answer]
Burn doesn't support that today.
